SpringProject/src/main/resources/db.migration/V2__addUsers.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE USERS (ID INT AUTO_INCREMENT PRIMARY KEY, USERID VARCHAR(45));
```

Comment:
This should be 'user', 'id' and 'user_id', all lower case, right @H-Ford @vsocolov-elsevier @mdykiert ?
What's the difference between id and user_id?